src/adapters/EasyRtcAdapter.js

```diff
@@ -31,16 +31,17 @@ class EasyRtcAdapter extends NoOpAdapter {
     this.easyrtc.joinRoom(roomName, null);
   }
 
-  // options: { datachannel: bool, audio: bool }
+  // options: { datachannel: bool, audio: bool, video: bool }
   setWebRtcOptions(options) {
     // this.easyrtc.enableDebug(true);
     this.easyrtc.enableDataChannels(options.datachannel);
 
-    this.easyrtc.enableVideo(false);
+    this.easyrtc.enableVideo(options.video);
     this.easyrtc.enableAudio(options.audio);
 
-    this.easyrtc.enableVideoReceive(false);
-    this.easyrtc.enableAudioReceive(true);
+    // TODO receive(audio|video) options ?
+    this.easyrtc.enableVideoReceive(options.video);
+    this.easyrtc.enableAudioReceive(options.audio);
```

Comment:
See https://github.com/networked-aframe/networked-aframe/commit/fccad7543f1255385ec0c9097044b39ab09540b2 were we changed from `options.audio` to `true`. So it seems you will introduce a regression here.

Reply:
ok will set default to true.